### PR TITLE
Add consensus stream documentation

### PR DESCRIPTION
**Pull request by movitto**
Documentation pertaining to the newly added consensus stream (rippled 1.4.0)

### CODE REVIEW

**Comment by mDuo13** [comment]
!rebuild

File: content/references/rippled-api/public-rippled-methods/subscription-methods/subscribe.md [hunk 1]
```diff
@@ -534,5 +535,24 @@ The format of an order book stream message is the same as that of [transaction s
...
+  "consensus": "accepted"
+}
+```
+
+The fields from a consensus stream message are as follows:
+
+| `Field`             | Type                      | Description                |
+|:--------------------|:--------------------------|:---------------------------|
+| `type`              | String                    | `consensusPhase` indicates this is from the consensus stream |
+| `consensus`         | String                    | The new consensus phase the server is in, one of 'open', 'establish', or 'accepted'.|
```
Comment (mDuo13) [inline content/references/rippled-api/public-rippled-methods/subscription-methods/subscribe.md]: Please change the description to the following:

> The new consensus phase the server is in. Possible values are `open`, `establish`, and `accepted`.

And please make sure there is a space before the closing `|`.

Also, I'd love to have descriptions of how those states correspond to the steps described in the [Consensus article](https://xrpl.org/consensus.html) but we can push this first and add those later.

File: content/references/rippled-api/public-rippled-methods/subscription-methods/subscribe.md [hunk 2]
```diff
@@ -534,5 +535,24 @@ The format of an order book stream message is the same as that of [transaction s
 | `transaction.owner_funds` | String | Numeric amount of the `TakerGets` currency that the `Account` sending this OfferCreate transaction has after executing this transaction. This does not check whether the currency amount is [frozen](freezes.html). |
 
 
+## Consensus Stream
+
+The `consensus` stream sends `consensusPhase` messages when [the consensus process](consensus.html) changes phase. The message contains the new phase of consensus the server is in.
```
Comment (mDuo13) [inline content/references/rippled-api/public-rippled-methods/subscription-methods/subscribe.md:542]: Please add `[New in: rippled 1.4.0][]` to the end of this line.

**Review by mDuo13 — CHANGES_REQUESTED** [review]
Thank you for adding this. I left a couple small suggestions I'd like to see applied before we merge this. Everything else looks great!

**Comment by movitto** [comment]
@mDuo13 made the requested changes. Since this is a small commit I didn't bother with the [FOLD] and just force pushed the single patch. Also to keep it consistent w/ the "Validations Stream" section above I put the "[New in: rippled 1.4.0][]" below the "Consensus Stream" header but above the description. If the original way you suggested is desired please just mention and I'll change it to that.

Thank you for reviewing

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**